airstory.php

```diff
@@ -16,6 +16,9 @@
 
 namespace Airstory;
 
+// Declare the canonical plugin version.
+define( 'AIRSTORY_VERSION', '0.1.0' );
```

Comment:
Yeah, I typically start everything off as a 0.1.0 but with the stability and amount of testing in this I'm comfortable with a 1.0.0.